### PR TITLE
feat: signup

### PR DESCRIPTION
**Pull request by thesudhansupanda**

### CODE REVIEW

File: templates/signup.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    <div class="badge">11</div>
+                </div>
+
+            </div>
+    </section> 
+
+    </nav>
+
+    <!-- <main class="main">
+        <div class="login">
```
Comment (shashidwivedi) [inline templates/signup.html:46]: Please remove this dead code

File: styles/main.css [hunk 2]
```diff
@@ -92,7 +92,7 @@ margin: .9rem  2rem;
     border: 1px solid;
 }
 
-.login-form .textbox{
+.login-form .textbox , .signup-form .textbox {
```
Comment (shashidwivedi) [inline styles/main.css:95]: Try to use prettier

**Review by shashidwivedi — APPROVED** [review]
Please see the comments. Rest looks good

File: templates/signup.html (hunk 1 above)
Comment (thesudhansupanda) [reply templates/signup.html]: done

File: styles/main.css (hunk 2 above)
Comment (thesudhansupanda) [reply styles/main.css:95]: done

**Pull request merged**